src/HttpClient/Guzzle.php

```diff
@@ -115,6 +115,10 @@ public function request($uri, $method = 'GET', $parameters = [], $headers = [],
     {
         $this->requestHeader = array_replace($this->requestHeader, (array) $headers);
 
+        if (($method == 'GET' || $method == 'DELETE') && (strpos($uri, '?') !== false)) {
```

Comment:
not sure - in fact query params can be in any kind of http method. Also, if there is such case - we probably just need to move query from the url to query as array.